packages/runtime/container-runtime/src/orderedClientElection.ts

```diff
@@ -216,6 +220,8 @@ export interface IOrderedClientElection extends IEventProvider<IOrderedClientEle
     readonly eligibleCount: number;
     /** Currently elected client. */
     readonly electedClient: ITrackedClient | undefined;
+    /** Currently elected parent, may be equal to electedClient */
```

Comment:
Same comment as above. It may be worth adding when this happens and what happens next.

Reply:
Sounds good.